### DISCUSSION TITLE
树莓派4b无法使用docker

### DISCUSSION THREAD

**Discussion by obeta** [natfrp/wiki#281] [使用问题] [answered]
zz@raspberrypi:~ $ sudo su
root@raspberrypi:/home/ly# docker pull natfrp/frpc
Using default tag: latest
latest: Pulling from natfrp/frpc
Digest: sha256:03145de8665241e5607851cba5fe47058c61e3be04a69c2efd6cee1a27facf7e
Status: Image is up to date for natfrp/frpc:latest
docker.io/natfrp/frpc:latest
root@raspberrypi:/home/ly# ip addr show docker0

5: docker0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default
    link/ether 02:42:77:53:da:51 brd ff:ff:ff:ff:ff:ff
    inet 172.17.0.1/16 brd 172.17.255.255 scope global docker0
       valid_lft forever preferred_lft forever
    inet6 fe80::42:77ff:fe53:da51/64 scope link
       valid_lft forever preferred_lft forever
root@raspberrypi:/home/ly# docker run \
-d \
--restart=always \
--pull=always \
--name=sakura1 \
natfrp/frpc \
-f pkjqsukv3eg4wlfld58ma3c63nmfcgn4 --remote_control 7139489
docker: invalid reference format.
See 'docker run --help'.
bash: --restart=always：未找到命令
bash: --pull=always：未找到命令
bash: --name=sakura1：未找到命令
bash: natfrp/frpc: 没有那个文件或目录
root@raspberrypi:/home/ly# docker run -d --restart=always --pull=always --name=sakura1 natfrp/frpc -f pkjqsukv3eg4wlfld58ma3c63nmfcgn4 --remote_control 7139489
latest: Pulling from natfrp/frpc
Digest: sha256:03145de8665241e5607851cba5fe47058c61e3be04a69c2efd6cee1a27facf7e
Status: Image is up to date for natfrp/frpc:latest
5cf58acaa450e1678f23860641bc47a982faed953a1abf71e57934d505a9f46b
root@raspberrypi:/home/ly#

172.17.0.1 我已开通了隧道

**Comment by obeta** [comment]
错误日志：

Bad fetch parameter
Bad fetch parameter
Bad fetch parameter
Bad fetch parameter
Bad fetch parameter
Bad fetch parameter
Bad fetch parameter

**Comment by fakeboboliu** [comment, marked as answer]
首先，请到 https://www.natfrp.com/user/profile 点击 重置访问密钥。

对于 Bad fetch parameter，是因为写错了参数，
如果您无法理解教程内容的话我们对于教程的易懂程度表示抱歉，
请到 [隧道列表](https://www.natfrp.com/tunnel/) 页面，点击横着的隧道条目后面竖向三个点的菜单按钮，然后点击“配置文件”按钮，
使用弹出的框的第一个文本框，内容形似 `-f xxxx:xxx` 的文本作为启动参数。
